packages/core/src/lib/tenant/tenant-setting/tenant-setting.service.ts

```
@@ -18,6 +18,173 @@ export class TenantSettingService extends TenantAwareCrudService<TenantSetting>
+		// Start with environment defaults
+		const resolvedSettings: Record<string, string> = { ...(envDefaults || {}) };
+
+		// Fetch global settings (tenantId = NULL)
+		let globalSettings: TenantSetting[];
+		switch (this.ormType) {
+			case MultiORMEnum.MikroORM:
+				const { where: globalWhere, mikroOptions: globalMikroOptions } =
+					parseTypeORMFindToMikroOrm<TenantSetting>({
+						where: { name: In(names), tenantId: IsNull() }
+					});
+				const globalItems = await this.mikroOrmRepository.find(globalWhere, globalMikroOptions);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)